tests/phpunit/tests/l10n/localeSwitcher.php

```diff
@@ -388,6 +394,40 @@ public function test_multiple_switches_to_site_locale_and_user_locale() {
 		$this->assertSame( 'This is a dummy plugin', $expected );
 	}
 
+	/**
+	 * @ticket 39210
+	 */
+	public function test_switch_reloads_translations_outside_wplang() {
+		global $wp_locale_switcher;
+
+		$locale_switcher = clone $wp_locale_switcher;
+
+		$wp_locale_switcher = new WP_Locale_Switcher();
+		$wp_locale_switcher->init();
```

Comment:
Without this the test will fail when the whole test suite (or the i18n group) runs. While we doe this in other tests this still feels wrong. Somehow the locale filter gets lost which `init()` will restore. @swissspidy Do you have any insights/ideas?

Reply:
Hmm odd. No clue at first glance.

In general, it feels like we should just instantiate `$wp_locale_switcher` in `setUp`, like the other globals.